src/shipment/controller.js

```diff
@@ -419,6 +428,30 @@ async function getPopulatedShipmentData(shipmentId = undefined) {
                       $expr: { $eq: ["$Items._id", "$$packedItemIds"] },
                     },
                   },
+                  { $lookup: {
+                    from: 'genOrders-v2',
+                    let: {
+                      orderNumber: '$OrderNumber',
+                      calcItemId: '$Items.calcItemId'
+                    },
+                    pipeline: [
+                      { $match: {
+                        $expr: { $eq: ['$orderNumber', '$$orderNumber'] },
+                      } },
+                      { $unwind: '$content.order.items' },
+                      { $match: {
+                        $expr: { $eq: [ { $toString: '$content.order.items._id' }, '$$calcItemId' ] }
+                      } },
+                      { $addFields: {
+                        unitRate: '$content.order.items.calculations.header.unitRate'
```

Comment:
use `finalUnitRate` instead of `unitRate`.
Final unit rate includes any discounts or upcharges and is what the customer actually pays for (i.e the reported value of the parts packaged)